purpleair-aqi.js

```diff
@@ -345,7 +345,7 @@ function computePM(sensorData) {
   const adj1 = Number.parseInt(sensorData.adj1, 10);
   const adj2 = Number.parseInt(sensorData.adj2, 10);
   const hum = Number.parseInt(sensorData.hum, 10);
-  const dataAverage = (adj1 + adj2) / 2;
+  const dataAverage = isNaN(adj2) ? adj1 : (adj1 + adj2) / 2;
```

Comment:
`parseInt` returns `NaN` (not a number) when given non-numeric string inputs, so we just check whether `adj2` even has a sensible value.